apps/allocations/package.json

```diff
@@ -35,6 +35,7 @@
     "@aragon/apps-vault": "4.1.0",
     "@aragon/os": "4.2.0",
     "@aragon/ui": "1.0.0-alpha.19",
+    "@tps/test-helpers": "0.0.1",
```

Comment:
Do we want this? See [discussion here](https://github.com/AutarkLabs/open-enterprise/pull/1290#discussion_r331277577).